<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_Obtencion">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Obtencion de robots</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_TamCod">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>10</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Tama√±o de codigos [cm]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Edit_NumCM"/>
        </item>
        <item>
         <widget class="QPushButton" name="boton_AceptarCM">
          <property name="text">
           <string>Aceptar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="boton_ObtenerFirst">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Obtener robots</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>Robot</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_ImgRobot">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_IdRobot">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>ID: 255</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="boton_AceptarRobot">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Aceptar Robot</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="boton_BorrarRobot">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Borrar Robot</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="boton_AceptarVRobots">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Aceptar configuracion</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="boton_CancelarRobots">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Cancelar configuracion</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_ObtencionIPs">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Obtencion de IPs</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="boton_ObtenerIPs">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Obtener IPs</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Tabla de robots</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTableWidget" name="tabla_robots">
           <property name="font">
            <font>
             <pointsize>7</pointsize>
            </font>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>100</number>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_Poses">
        <property name="text">
         <string>Poses de robots</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
